current/docs/development-best-practices.adoc

```diff
@@ -185,25 +185,41 @@ The following snippet should work when using a library with the groupId `org.som
 
 [source, xml]
 ----
-<plugin>
-    <groupId>org.apache.maven.plugins</groupId>
-    <artifactId>maven-shade-plugin</artifactId>
-    <version>2.3</version>
-    <configuration>
-        <!-- custom shade configuration -->
-        <artifactSet>
-            <includes>
-                <include>org.some.library:custom-artefact-id</include>
-            </includes>
-        </artifactSet>
-        <relocations>
-            <relocation>
-                <pattern>org.some.library</pattern>
-                <shadedPattern>org.some.library.new.pakage.name.shade</shadedPattern>
-            </relocation>
-        </relocations>
-    </configuration>
-</plugin>
+<dependencies>
+    <dependency>
+        <groupId>org.some.library</groupId>
+        <artifactId>custom-artefact-id</artifactId>
```

Comment:
it should be "artifact" instead of "artefact". We already had the typo in the shading configuration.